qutebrowser/browser/hints.py

```diff
@@ -748,6 +760,13 @@ def start(self, rapid=False, group=webelem.Group.all, target=Target.normal,
         self._context.tab.find_all_elements(selector, self._start_cb,
                                             only_visible=True)
 
+    def current_mode(self):
+        """ Returns the currently active hinting mode (or None otherwise). """
+        if not self._context or not self._context.hint_mode:
+            return None
+
+        return self._context.hint_mode
```

Comment:
I usually don't use newlines before a `return` if the function is short enough, but it's probably purely a personal preference - no strong feelings either way.

> An explicit check is more robust and only costs one line.

Fair enough!